app/src/main/java/com/internship/move/feature/authentification/login/LoginFragment.kt

```diff
@@ -15,12 +18,50 @@ class LoginFragment : Fragment(R.layout.fragment_login) {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
-        binding.loginFragmentRegisterBtn.setOnClickListener {
-            findNavController().navigate(LoginFragmentDirections.actionLoginFragmentToRegisterFragment())
+        initClickableTV()
+        initListeners()
+    }
+
+    private fun initListeners() {
+        binding.emailTIL.isEndIconVisible = false
+        binding.passwordTIL.isEndIconVisible = false
+
+        binding.emailTIL.setEndIconOnClickListener {
+            binding.emailTIET.text?.clear()
         }
 
-        binding.loginFragmentHomeBtn.setOnClickListener {
+        binding.loginBtn.setOnClickListener {
             findNavController().navigate(LoginFragmentDirections.actionLoginFragmentToHomeGraph())
         }
+
+        binding.emailTIET.doOnTextChanged { text, _, _, _ ->
+            binding.emailTIL.isEndIconVisible = text?.isNotEmpty() == true
+            changeBtnState(
+                binding.emailTIET.text?.isNotEmpty() ?: false,
+                binding.passwordTIET.text?.isNotEmpty() ?: false
+            )
+        }
+
+        binding.passwordTIET.doOnTextChanged { text, _, _, _ ->
+            binding.passwordTIL.isEndIconVisible = text?.isNotEmpty() == true
+            changeBtnState(
+                binding.emailTIET.text?.isNotEmpty() ?: false,
+                binding.passwordTIET.text?.isNotEmpty() ?: false
+            )
+        }
+    }
+
+    private fun initClickableTV() {
+        binding.forgotPasswordTV.makeLinks(getString(R.string.authentication_login_forgot_password), action = {
```

Comment:
Since this is a lambda function, you can simplify these calls like this
```
binding.forgotPasswordTV.makeLinks(getString(R.string.authentication_login_forgot_password)) {
    findNavController().navigate(LoginFragmentDirections.actionLoginFragmentToForgotPasswordFragment())
}
```